tests/python/gaia-ui-tests/gaiatest/apps/gallery/regions/crop_view.py

```diff
@@ -19,14 +19,16 @@
 class CropView(Base):
     _src = 'app://gallery.gaiamobile.org/index.html#pick'
 
-    _crop_view_locator = (By.ID, 'crop-view')
     _crop_done_button_locator = (By.ID, 'crop-done-button')
+    _edit_preview_canvas_locator = (By.ID, 'edit-preview-canvas')
 
     def __init__(self, marionette):
         Base.__init__(self, marionette)
         Wait(self.marionette).until(expected.element_displayed(
             Wait(self.marionette).until(expected.element_present(
-                *self._crop_view_locator))))
+                *self._edit_preview_canvas_locator))))
+        import time
+        time.sleep(2)
```

Comment:
Why the sleep here?

Can you add a comment to explain why we need a hardcoded sleep